src/services/identity/DeploymentsService.ts

```diff
@@ -249,9 +254,22 @@ class DeploymentsService {
       return errAsync(
         new NotFoundError(`Deployment for site ${repoName} does not exist`)
       )
-    const { id, hostingId: appId } = deploymentInfo
+    const {
+      id,
+      hostingId: appId,
+      stagingLiteHostingId: stagingLiteId,
+    } = deploymentInfo
 
-    if (!enablePassword) return this.deletePassword(appId, id)
+    if (!enablePassword) {
+      const stagingRes = await this.deletePassword(appId, id, false)
+      if (!stagingLiteId || stagingRes.isErr()) return stagingRes
+      const stagingLiteRes = await this.deletePassword(stagingLiteId, id, true)
+      if (stagingLiteRes.isErr())
+        logger.error(
+          `Privatisation adjustment failed for ${repoName} - requires manual fixing of inconsistent state`
+        )
```

Comment:
do we have an alarm for this? otherwise, we won't know when this occurs until someone feedsback to us, rendering this log moot

Reply:
Hey yup the alarm's already been set up!
https://ap-southeast-1.console.aws.amazon.com/cloudwatch/home?region=ap-southeast-1#alarmsV2:alarm/cms-backend-prod-node18-privatisation-inconsistency-error?~(search~'privatisation)